packages/dialtone-vue2/recipes/leftbar/contact_centers_row/contact_centers_row.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <div
+    :class="leftbarContactCentersRowClasses"
+    data-qa="dt-leftbar-row"
```

Comment:
This should be the name of the component to avoid conflicts later and make testing easier, e.g:

```suggestion
    data-qa="dt-recipe-contact-centers-row"
```